.travis.yml

```diff
@@ -0,0 +1,3 @@
+language: node_js
+node_js:
+  - "5"
```

Comment:
according to package.json, it's currently running 5.9.1. Can change to 6 though, shouldn't be a problem.